src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -143,9 +143,9 @@ class PaymentInformation extends React.Component {
 
   renderSetupButton(label, gaProfileSection) {
     return (
-      <a
+      <button className="va-button-link"
```

Comment:
ESLint is gonna complain about this line. You can see this by running `yarn lint:js`. You can also set up VSCode to use the ESLint extension and show (and sometimes fix) the errors right in the editor.

Reply:
Just a straight brain fart. I didn't have any vscode extensions installed. I just installed React Food Truck and was able to start seeing eslint errors.